Modules/symtablemodule.c

```diff
@@ -58,7 +58,6 @@ _symtable_symtable_impl(PyObject *module, PyObject *source,
     }
     t = (PyObject *)st->st_top;
     Py_INCREF(t);
-    PyMem_Free((void *)st->st_future);
```

Comment:
Aren't you leaking `st_future`?

Reply:
This seems like something we caught while testing, though when you create a symbol table from a string (for python interface), then it doesn't actually set the futures. And while checking that, I noticed that this free was unnecessary and previously had no effect. Now it is trying to deallocate something that we didn't allocate in this function in the first place.